### ISSUE TITLE
Jest testing componentWillMount renaming warning

### ISSUE THREAD

**Issue by Lukortech** [americanexpress/react-albus#68]
Hi! Sorry to bother but is there a chance for update for Steps and Wizard so that those components will use `UNSAFE_ `?

While testing I am getting this warning which is just a small reminder of how newer version of react handles lifecycles if I am correct.

   ``` console.warn node_modules/react-dom/cjs/react-dom.development.js:88
      Warning: componentWillMount has been renamed, and is not recommended for use. See https://fb.me/react-unsafe-component-lifecycles for details.
      
      * Move code with side effects to componentDidMount, and set initial state in the constructor.
      * Rename componentWillMount to UNSAFE_componentWillMount to suppress this warning in non-strict mode. In React 17.x, only the UNSAFE_ name will work. To rename all deprecated lifecycles to their new names, you can run `npx react-codemod rename-unsafe-lifecycles` in your project source folder.
      
      Please update the following components: Steps, Wizard```

**Comment by ryhinchey** [comment]
This issue can be closed.  `componentWillMount` isn't being used in the library any more

**Comment by Lukortech** [comment]
Thank you for update! :)

**Comment by aminace-qh** [comment]
@ryhinchey 

> This issue can be closed. `componentWillMount` isn't being used in the library any more

Is this change released with the latest version of the lib (v2.0.0)?
